frappe/api/v2.py

```diff
@@ -92,7 +117,7 @@ def count(doctype: str) -> int:
 def create_doc(doctype: str):
 	data = frappe.form_dict
 	data.pop("doctype", None)
-	return frappe.new_doc(doctype, **data).insert()
+	return frappe.new_doc(doctype, **data).insert().as_dict()
```

Comment:
Is this required? :thinking: docs get serialized to JSON in the end anyway.

Reply:
Document object gets serialized to JSON but skips fields with `null` values.